NwPluginAPI/Core/Doors/FacilityGate.cs

```diff
@@ -22,6 +22,11 @@ public class FacilityGate : FacilityDoor
 		/// </summary>
 		public new static int Count => List.Count;
 
+		/// <summary>
+		/// Gets gate name.
+		/// </summary>
+		public string Name => OriginalObject.name;
```

Comment:
Is this needed? `FacilityGate` inherits from `FacilityDoor` and you already added `Name` there.

Reply:
You are right

Reply:
Fixed in commit [da48294](https://github.com/northwood-studios/NwPluginAPI/pull/161/commits/da482942d286f980f8287b966f4c54babb84248a)